website/docs/components/alert/02--component-api.md

```diff
@@ -7,12 +7,12 @@ section: component-api
 
 Here is the API for the component:
 
-<dl class="dummy-component-props" aria-labelledby="component-api-alert"><dt>type <code>enum</code> <strong class="required">required</strong></dt><dd><p>Sets the type of alert.</p><p>Acceptable values:</p><ol><li>page</li><li>inline</li><li>compact</li></ol></dd><dt>color <code>enum</code></dt><dd><p>Sets the color scheme for <code class="dummy-code">background</code>, <code class="dummy-code">border</code>, <code class="dummy-code">title</code>, and <code class="dummy-code">description</code>, which <strong>cannot</strong> be overridden. <code class="dummy-code">color</code> results in a default <code class="dummy-code">icon</code>, which <strong>can</strong> be overridden.</p><p>Acceptable values:</p><ol><li class="default">neutral</li><li>highlight</li><li>success</li><li>warning</li><li>critical</li></ol></dd><dt>icon <code>string | false</code></dt><dd><p>Override the default <code class="dummy-code">icon</code> name, which is determined by the <code class="dummy-code">color</code> argument.</p><p>Acceptable values: any <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">Flight</a> icon name or pass <code class="dummy-code">false</code> for no icon.</p></dd><dt>onDismiss <code>function</code></dt><dd><p>The alert can be dismissed by the user. When a function is passed, the "dismiss" button is displayed.</p></dd><dt>...attributes</dt><dd><p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p></dd></dl>
+<Doc::ComponentApi as |C|><C.Property @name="type" @required="true" @type="enum" @value="page inline compact">Sets the type of alert.</C.Property><C.Property @name="color" @type="enum" @value="neutral highlight success warning critical" @default="neutral">Sets the color scheme for `background`, `border`, `title`, and `description`, which **cannot** be overridden. `color` results in a default `icon`, which **can** be overridden.</C.Property><C.Property @name="icon" @type="string | false" @value="–">Override the default `icon` name, which is determined by the `color` argument. [Flight](https://flight-hashicorp.vercel.app/) icon name or pass `false` for no icon.</C.Property><C.Property @name="onDismiss" @type="function" @value="–">The alert can be dismissed by the user. When a function is passed, the "dismiss" button is displayed.</C.Property><C.Property @name="...attributes" @type="–" @value="–">`...attributes` spreading is supported on this component.</C.Property></Doc::ComponentApi>
```

Comment:
My understanding is that for how `turndown` works they need to be inline, not multiline/nested right?
Not in this PR but in the follow-up, if you can't find a way around this, maybe you could consider a post-processing script that using a simple regex takes this text block, and then pass it to a prettier function* and let it do the job for you. just an idea

\* _prettier can be imported as module; I'm sure you already know it, but if you don't... see for example how it's done [here](https://github.com/hashicorp/design-system/blob/main/packages/flight-icons/scripts/build-parts/getCssForIconAnimation.ts) or [here](https://github.com/hashicorp/design-system/blob/main/packages/flight-icons/scripts/build-parts/generateBundleSVGReact.ts)_